setup.py

```diff
@@ -32,12 +33,19 @@
     author_email='jan.freyberg@gmail.com',
     packages=['niwidgets'],
     keywords=['widgets', 'neuroimaging'],
-    install_requires=['ipywidgets', 'nilearn', 'nibabel', 'ipyvolume'],
+    install_requires=['ipywidgets', 'nilearn', 'nibabel', 'ipyvolume',
```

Comment:
Is nilearn a real dependency? I don't see it imported in any of the code. Only in examples.

Reply:
This is true, yep.